packages/web-components/fast-foundation/src/menu/stories/menu.register.ts

```diff
@@ -26,6 +26,8 @@ const styles = css`
     :host([slot="submenu"]) {
         margin: 0 calc(var(--design-unit) * 1px);
         width: max-content;
+        max-height: 94vh;
+        overflow-y: auto;
```

Comment:
Is this to test the given scenario or does the change require code changes such as this? Just curious so we cna communicate it effectively

Reply:
This enables the "scroll bars show up when the sub-menu is taller than the viewport.  The old approach will continue to work but the sub-menu can extend beyond the viewport if there is not enough room.